task/build-paketo-builder-oci-ta/0.2/build-paketo-builder-oci-ta.yaml

```
@@ -144,6 +144,23 @@ spec:
         set -eu
         set -o pipefail
 
+        echo "[$(date --utc -Ins)] Validate context path"
+
+        source_dir_path=$(realpath "$SOURCE_CODE_DIR")
+        context_dir_path=$(realpath "$SOURCE_CODE_DIR/$CONTEXT")
+
+        case "$context_dir_path" in
+          "$source_dir_path"*)
```

```suggestion
          "$source_dir_path/"*)
```

Like this I think

`/tmp/source-foo` is not a subpath of `/tmp/source` but the current check would think it is

---

Thank you for catching this edge case. Updated. Also, if the context is empty, it defaults to '.'